packages/contracts-bedrock/scripts/Artifacts.s.sol

```diff
@@ -81,6 +85,28 @@ abstract contract Artifacts {
         }
     }
 
+    function hasSuffix(string memory _fullString, string memory _suffix) public pure returns (bool) {
```

Comment:
From @tynes: We should see if solady has a helper here for this so we don't need to implement/test our own code

Reply:
https://github.com/Vectorized/solady/blob/1f43cc8005cc3b3c8361dd7dbdd2cdeaf0f99e66/src/utils/LibString.sol#L678